libfwupd/fwupd-enums.h

```diff
@@ -1077,6 +1077,7 @@ typedef enum {
  * @FWUPD_VERSION_FORMAT_SURFACE:		Microsoft Surface 8b.16b.8b
  * @FWUPD_VERSION_FORMAT_DELL_BIOS:		Dell BIOS BB.CC.DD style
  * @FWUPD_VERSION_FORMAT_HEX:			Hexadecimal 0xAABCCDD style
+ * @FWUPD_VERSION_FORMAT_TRIPLET_MINOR:		CFU-style AA.BBCC.DD version numbers
```

Comment:
Is this not the same as `FWUPD_VERSION_FORMAT_SURFACE`?

Reply:
Ohh yes! Good catch, thanks!